lib/graph_records.rb

```diff
@@ -1,23 +1,206 @@
 # frozen_string_literal: true
 
+require "digest"
+
 # Spit out a Mermaid-style graph of records.
 #
 # Usage:
 #  graph = GraphRecords.new.graph(patients: [patient])
 #  puts graph
 #
 class GraphRecords
+  # TODO: put all these constants elsewhere?
+  BOX_STYLES = %w[
+    fill:#e6194B,color:white
+    fill:#3cb44b,color:white
+    fill:#ffe119,color:black
+    fill:#4363d8,color:white
+    fill:#f58231,color:white
+    fill:#911eb4,color:white
+    fill:#42d4f4,color:black
+    fill:#f032e6,color:white
+    fill:#bfef45,color:black
+    fill:#fabed4,color:black
+    fill:#469990,color:white
+    fill:#dcbeff,color:black
+    fill:#9A6324,color:white
+    fill:#fffac8,color:black
+    fill:#800000,color:white
+    fill:#aaffc3,color:black
+    fill:#808000,color:white
+    fill:#ffd8b1,color:black
+    fill:#000075,color:white
+    fill:#a9a9a9,color:white
+    fill:#ffffff,color:black
+    fill:#000000,color:white
+  ].freeze
+
+  DEFAULT_NODE_ORDER = %i[
```

Comment:
As part of the comment above on `Inspect::Graph::GraphController#safe_get_primary_type`, it may simplify other parts of the code if these are strings:

```suggestion
  DEFAULT_NODE_ORDER = %w[
```